src/LuaMultiThreaded2.cpp

```diff
@@ -73,23 +73,29 @@ extern "C" static int threadNew(LuaState * aState)
 {
 	static std::recursive_mutex mtx;
 	std::scoped_lock lock(mtx);
+	auto args = lua_gettop(aState);
```

Comment:
```suggestion
	auto numParams = lua_gettop(aState);
```
This way it's clear that it's not the actual args / params.